homeassistant/components/cloud/__init__.py

```diff
@@ -176,6 +177,23 @@ def async_active_subscription(hass: HomeAssistant) -> bool:
     return async_is_logged_in(hass) and not hass.data[DOMAIN].subscription_expired
 
 
+@bind_hass
```

Comment:
I don't think we should add `bind_hass` to new code. It's legacy and we're trying to get rid of it.